doc/classes/NavigationServer3D.xml

```diff
@@ -342,6 +342,13 @@
 				Returns information about the current state of the NavigationServer. See [enum ProcessInfo] for a list of available states.
 			</description>
 		</method>
+		<method name="is_baking" qualifiers="const">
```

Comment:
comment from @smix8 via chat: `for the server api make the function name more explicit to what it does, e.g. is_navigation_mesh_baking()`